### PR TITLE
Portfolio Page - setup and mobile first

### PR DESCRIPTION
**Pull request by jossi30**
added a navigation header and a headline section right after the header.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+.intro {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400px;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+}
```
Comment (franclobo) [inline style.css:73]: - [ ] Kindly add a distance between the title and the paragraph to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| image | image |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    <body>
+        <header class="container">
+           <nav class="header-nav">
+               <div class="logo">jossi30</div>
+               <div class="humberger">
+                   <span class="bar"></span>
+                   <span class="bar"></span>
+                   <span class="bar"></span>
+               </div>
+           </nav>
```
Comment (franclobo) [inline index.html:20]: - [ ] Please 🙏🏽 close the header tag after the nav tag, for the best practices of HTML&CSS, kindly [check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @jossi30, 

# Required Changes ♻️

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,275 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
```
Comment (Tufahel) [inline README.md:111]: ***[OPTIONAL]*** `Live demo link` means the live version of your website that can be accessed from anywhere in the world. You have added the wrong link. 
You can consider adding the correct link as it is professional, and the viewer can easily find your website's online version from Readme. Here is a [FreecodeCamp Link](https://www.freecodecamp.org/news/publish-your-website-netlify-github/) for reference.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+          </header> 
+           <section class="welcome">
+            <div class="hello">Hey There. I’m <br> Yosief</div>
```
Comment (Tufahel) [inline index.html:23]: ***[OPTIONAL]*** Kindly consider using `h1` tag for `Hey There. I'm Yosief`. Though it is one of the core requirements, I have made it optional, and `sorry for that the previous reviewer missed it`.  
The reason behind this core requirement is: ***If you want that google (and other search engines) know about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.***

File: index.html [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+            <div class="detail">I’m a Software <br>Developer</div>
+            <div class="intro"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
```
Comment (Tufahel) [inline index.html:25]: ***[OPTIONAL]*** Kindly use specific tags for specific sections.
Such as: 
- For `I'm a Software Developer you could use `h2` or `h3` tags.
- For a paragraph, it's better to use a `p` tag inside the `div` tag.

Here is a [reference](https://stackoverflow.com/questions/28529390/ok-to-have-text-in-div-without-paragraph-tag) from Stackoverflow about why it's better to use a specific tag for a specific section. You can consider checking it.
`Though it is also an important issue, Kindly fix it before merging.`

**Review by Tufahel — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @jossi30  

Your project is complete now!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## To Highlight 👏🏻

:heavy_check_mark: No linter errors.
:heavy_check_mark: Implemented the previous reviewer's suggestion.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**